src/OpenLoco/Title.cpp

```diff
@@ -52,7 +52,7 @@ namespace OpenLoco::Title
 
     // Explicit deduction guide (not needed as of C++20)
     template<class... Ts>
-    overloaded(Ts...)->overloaded<Ts...>;
+    overloaded(Ts...) -> overloaded<Ts...>;
```

Comment:
This fails clang-format due to CI using an older version

Reply:
this gets me every time I touch this file haha. will fix it soon